File: app/src/main/java/com/example/softballapp/ObjectBox.java
package com.example.softballapp;
import android.content.Context;
import io.objectbox.BoxStore;
public class ObjectBox {

    private static BoxStore boxStore;
    public static void init(Context context)
    {
        boxStore = MyObjectBox.builder().androidContext(this).build();
        //test
    }
}

File: app/src/main/java/com/example/softballapp/ObjectBox.kt
package com.example.softballapp

import android.content.Context
import android.util.Log
import io.objectbox.BoxStore

/**
 * Singleton to keep BoxStore reference.
 */
object ObjectBox {

    lateinit var boxStore: BoxStore
        private set

    fun init(context: Context) {
        boxStore = MyObjectBox.builder().androidContext(context.applicationContext).build()

    }

}
